packages/dx-react-grid/docs/guides/data-types.md

```diff
@@ -20,7 +20,7 @@ const rows = [
 ];
 const columns = [
   { name: 'product', title: 'Product' },
-  { name: 'amount', title: 'Sale Amount', dataType: 'currency' },
+  { name: 'price', title: 'Sale Price', dataType: 'currency' },
```

Comment:
Not sure that this change should be merged into master. It will be better to revert this line.

Reply:
but the row price property doesn't match column name. Sale Amount or price can't display on the grid.

![capture](https://user-images.githubusercontent.com/4963061/33259229-d6889a88-d38e-11e7-94b0-008d87cd584a.JPG)

Reply:
Oh. I see. My fault. Let's rename the `price` field to `amount` in the `rows` constant.